spec/support/idv_examples/confirmation_step.rb

```diff
@@ -0,0 +1,61 @@
+shared_examples 'idv confirmation step' do |sp|
+  context 'after choosing to verify by letter' do
+    before do
+      start_idv_from_sp(sp)
+      complete_idv_steps_with_usps_before_confirmation_step
+    end
+
+    it 'redirects to the come back later url then to the sp or account' do
+      click_acknowledge_personal_key
+
+      # SAML test SP does not have a return URL, so it does not have a link
+      # back to the SP
```

Comment:
Should we create a new test SAML SP that does have a return url so we can test that scenario?

Reply:
Although I wonder if our existing view specs cover this already. What does a feature spec give us in this case? https://github.com/18F/identity-idp/blob/master/spec/views/verify/come_back_later/show.html.slim_spec.rb

Reply:
Yeah, I'm fine tearing down some of the logic that is covered by the view spec. I also think we could tear down the spec entirely since this is covered by the sp handoff spec.

Reply:
Actually, wait, this is not covered by the sp handoff spec because that is not aware of the user's verification method...

Reply:
Okay, I simplified this one a bit to focus on the path through the step and removed things covered by the view spec.